### ISSUE TITLE
numpy is not providing  f2py binary 

### ISSUE THREAD

**Issue by ajayk** [wolfi-dev/os#22668]
currently https://github.com/wolfi-dev/os/pull/22546  update is failing due to f2py binary not provided by numpy other distributions seems to provide the f2py binary as apart of the numpy 

```
scipy/meson.build:147:7: ERROR: Program 'f2py' not found or not executable
```

**Comment by pnasrat** [comment]
I'll investigate this

**Comment by pnasrat** [comment]
```
python3.12  -m numpy.f2py -v
1.26.4
```

**Comment by pnasrat** [comment]
wrapper script seems missing but module is installed

**Comment by pnasrat** [comment]
Fix in #22546
